src/System.AppContext/src/System.AppContext.builds

```diff
@@ -1,14 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
-  <ItemGroup Condition="'$(TargetsWindows)' == 'true'">
+  <ItemGroup>
     <Project Include="System.AppContext.csproj">
       <AdditionalProperties>TargetGroup=</AdditionalProperties>
     </Project>
-    <Project Include="System.AppContext.csproj">
+    <Project Include="System.AppContext.csproj" Condition="'$(TargetsWindows)' == 'true'">
       <AdditionalProperties>TargetGroup=netcore50</AdditionalProperties>
     </Project>
-    <Project Include="System.AppContext.csproj">
+    <Project Include="System.AppContext.csproj" Condition="'$(TargetsWindows)' == 'true'">
+      <AdditionalProperties>TargetGroup=net46</AdditionalProperties>
+    </Project>
+    <Project Include="facade\System.AppContext.csproj">
```

Comment:
Looks like this entry is redundant with the first one. I suspect we should simply eliminate this from your generation as I think we already did it properly in the root project.